assets/app/actions/routeActions.js

```diff
@@ -0,0 +1,12 @@
+import { navigationTypes } from '../constants';
+import store from '../store';
+
+export default {
+  redirect(path) {
+    store.dispatch({
+      type: navigationTypes.UPDATE_ROUTER,
```

Comment:
Are you saying this is an example if the type of action creator you'd like to see, or that this action creator should be changed?